.github/workflows/build-push.yml

```diff
@@ -75,8 +73,9 @@ jobs:
           IMAGE_NAME: ${{ matrix.image }}
           BUILDER_NAME: ${{ matrix.builder }}
         run: |
-          make ${IMAGE_NAME} BUILDER=${BUILDER_NAME} REG=ghcr.io/intel/ TAG=devel
-        if: ${{ github.event_name == 'push' && github.ref_name == 'main' && matrix.builder == 'docker' }}
+          make ${IMAGE_NAME} BUILDER=${BUILDER_NAME} REG=ghcr.io/intel/
+        if: ${{ github.event_name == 'push' && matrix.builder == 'docker' }}
       - run: |
-          docker push ghcr.io/intel/${{ matrix.image }}:devel
-        if: ${{ github.event_name == 'push' && github.ref_name == 'main' && matrix.builder == 'docker' }}
+          eval $(make -p -n |grep -e '^TAG = ' |tr -d ' ')
+          docker push ghcr.io/intel/${{ matrix.image }}:$TAG
```

Comment:
can `make` and `docker push` use the same `$TAG` which would be based on

```
on:
  push:
    tags:
       - 'v[0-9]+.[0-9]+.[0-9]+'
```

or `devel` otherwise?

Reply:
In current approach I wanted to rely on existing workflow style where Makefile defines TAG. What you propose is that push event based on tag change triggers a job where we get TAG value from GH parameter, and we set TAG outside of make. Yes I think that may work as well. Then it will be possible to stop editing TAG value in Makefile in various branches?